src/mainwindow.cpp

```diff
@@ -168,7 +168,12 @@ void MainWindow::run()
 
     for (int i = 0; i < testcases->count(); ++i)
     {
-        if (!testcases->input(i).trimmed().isEmpty() && testcases->isShow(i))
+        if (testcases->input(i).trimmed().isEmpty())
+        {
+            if (SettingsHelper::isAlwaysRun())
```

Comment:
```suggestion
            if (SettingsHelper::isRunOnEmptyTestCases())
```

Member might be different that this.